examples/index.html

```diff
@@ -21,14 +21,7 @@
     </style>
   </head>
   <body>
-    <div class="Dashboard-canvas js-dashboard">
-      <div class="Dashboard-mapWrapper Dashboard-mapWrapper--margin">
-        <div class="Map-canvas">
-          <div class="Map" id="map"></div>
-        </div>
-      </div>
-      <div class="Dashboard-belowMap js-dashboard-belowMap"></div>
-    </div>
+    <div class="Dashboard-canvas" id="dashboard"></div>
```

Comment:
Yep, already thought about that :point_right: https://github.com/CartoDB/cartodb.js/pull/887/files#diff-d93a93ee72a553cf85b1bcf4ecb61142R20